setup.py

```diff
@@ -8,13 +8,11 @@
     install_requires=[
         # client:
         "requests",
-        "urllib3<2.0.0, >=1.26.0",
+        "urllib3>=1.26.0",
         "pydantic>=2.0.0",
-        "typing-extensions>=4.5.0",
         "packaging"
     ],
     tests_require=[
-        "pytest",
         "tox"
     ],
     name="marqo",
```

Comment:
Do you want to upgrade the version here to publish it to Pypi?

Reply:
will do. @wanliAlex , this should be a new minor version, right?

Reply:
Yes, sounds like a patch version.